Examples/Modules/laser_injection_from_file/inputs.3d_test_txye

```diff
@@ -0,0 +1,47 @@
+#################################
+####### GENERAL PARAMETERS ######
+#################################
+max_step = 150
+amr.n_cell =  352 128 128
```

Comment:
The longitudinal resolution can probably not be further reduced, but what about the transverse resolution?
```suggestion
amr.n_cell =  352 64 64
```